src/wp-content/themes/twentytwentyone/assets/js/primary-navigation.js

```diff
@@ -198,7 +199,7 @@ function twentytwentyoneExpandSubMenu( el ) { // jshint ignore:line
 		} );
 	};
 
-	window.addEventListener( 'load', function() {
+	document.addEventListener( 'DOMContentLoaded', function() {
```

Comment:
The `load` event fires after all images load, meaning the menu could be uninitialized for a while after the DOM has loaded.